### PR TITLE
Fix: deny accept request when namespace is terminating.

### PR DESCRIPTION
**Pull request by xuezhaojun**
Related to: https://github.com/open-cluster-management-io/registration-operator/pull/342

### CODE REVIEW

File: pkg/webhook/v1/managedcluster_validating.go [hunk 1]
```diff
@@ -173,6 +184,29 @@ func (r *ManagedClusterWebhook) allowUpdateAcceptField(clusterName string, userI
 	return nil
 }
 
+// validateClusterNamespace checks the cluster namespace, if the namespace is terminating, reject the accept request.
+func (r *ManagedClusterWebhook) validateAcceptByClusterNamespace(clusterName string) error {
+	clusterNamespace, err := r.kubeClient.CoreV1().Namespaces().Get(context.TODO(), clusterName, metav1.GetOptions{})
+	if err != nil {
```
Comment (qiujian16) [inline pkg/webhook/v1/managedcluster_validating.go:193]:
```go
if apierrors.IsNotFound(err) {
	return nil
}
if err != nil {
  return ...
}
```

Comment (xuezhaojun) [reply pkg/webhook/v1/managedcluster_validating.go:193]: Fixed.

Comment (xuezhaojun) [reply pkg/webhook/v1/managedcluster_validating.go:193]: Please take another look when you have time! Thanks!

**Pull request merged**